scaffolds/src/main/kotlin/com/decathlon/vitamin/compose/scaffolds/VitaminScaffolds.kt

```diff
@@ -0,0 +1,152 @@
+        title: String,
+        modifier: Modifier = Modifier,
+        topBarConfiguration: TopBarConfiguration = TopBarConfiguration.Primary(),
+        topActionsUi: TopActionsUi = TopActionsUi(),
+        tabActionsUi: TabActionsUi = TabActionsUi(),
+        bottomActionsUi: BottomActionsUi = BottomActionsUi(),
+        fabAction: FabActionUi? = null,
+        tabSelectedIndex: Int? = null,
+        bottomRouteSelected: String? = null,
+        onTopActionClicked: (TopActionUi) -> Unit = {},
+        onTabClicked: (TabActionUi) -> Unit = {},
+        onBottomActionClicked: (BottomActionUi) -> Unit = {},
+        onFabActionClicked: (FabActionUi) -> Unit = {},
+        onSearchValueChanged: (String) -> Unit = {},
+        content: @Composable (PaddingValues) -> Unit
+    ) {
+        val expandedMenu = remember { mutableStateOf(false) }
+        Scaffold(
+            modifier = modifier,
+            topBar = {
+                VitaminTopBar(
+                    title = title,
+                    topBarConfiguration = topBarConfiguration,
+                    onSearchValueChanged = onSearchValueChanged,
+                    topActionsUi = topActionsUi,
+                    onTopActionClicked = onTopActionClicked,
+                    expandedMenu = expandedMenu,
+                    tabActionsUi = tabActionsUi,
+                    tabSelectedIndex = tabSelectedIndex,
+                    onTabClicked = onTabClicked
+                )
+            },
+            bottomBar = {
+                if (bottomActionsUi.actions.isNotEmpty()) {
+                    VitaminBottomNavigations.Primary(
+                        actions = bottomActionsUi.actions.map { action ->
+                            val hasSelectedRoutes =
+                                action.selectedRoutes.find { it == bottomRouteSelected } != null
+                            val selected = hasSelectedRoutes || action.route == bottomRouteSelected
+                            val icon = if (selected) action.iconSelected else action.icon
+                            SelectedActionItem(
+                                icon = rememberVectorPainter(image = icon),
+                                contentDescription = action.contentDescription,
+                                text = action.label,
+                                selected = selected,
+                                onClick = {
+                                    onBottomActionClicked(action)
+                                    return@SelectedActionItem true
+                                }
+                            )
+                        }
+                    )
+                }
+            },
+            backgroundColor = VitaminTheme.colors.vtmnBackgroundSecondary,
+            contentColor = VitaminTheme.colors.vtmnContentPrimary,
+            content = content,
+            floatingActionButton = {
+                if (fabAction != null) {
+                    VitaminFabs.Primary(
```

Comment:
Here you don't allow Scaffold user to change Fab properties like `sizes`, `colors`, `enabled`. Is it because you don't see the use case ?

Reply:
@ManonPolle 
I can imagine why one would want to enable/disable or show/hide the FAB but aren't `sizes` and `colors` already defined by the Design System?

Reply:
Yes there are. We provide Decathlon/Vitamin default `sizes` and `colors` but we let these params customizable to answer specific needs. 
If we allow this customization when implementing the component, we could also allow it inside the scaffold component.